design/decisions/why-license/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Why choose MIT License"
+description: |
+  Open and permissive licensing of copyrighted content is a way to increase 
+  use of a software product, ease collaboration and re-use, and share
+  intellectual ideas with others. The MIT License is a very permissive 
+  copyright license.
+author: "Kris Beicher"
+date: "2023-01-12"
+categories:
+  - licensing
+  - copyright
+---
+
+For a project like Seedcase to fully succeed in our stated mission, a
+critical key to that is what our copyright license is. Without a
+license, other users are not allowed to (easily) (re-)use, modify,
+contribute, or enhance our software because of copyright reasons. Since
+our mission isn't to just build a product, it's also to develop a
+creative community around Seedcase and the topics we work on. We want
+people to be able to freely and smoothly contribute enhancements and
+other improvements that can be incorporated into future releases of
+Seedcase. It is also our intention to make the Seedcase software
+available to commercial enterprises, which means that we need to
+carefully consider which license we adopt.
+
+Below is a walk-through of the different types of licenses that we
+believe we will need and a discussion on the reasons we make the choices
+that we do.
+
+## Types of licenses
+
+The Open Source Inititative [approves](https://opensource.org/licenses)
+a specific set of licenses that determine whether a project can be
+called "open source". These licenses are our starting point on deciding
+which license to use and how permissive we want it to be, in terms of
+who can work on our code and how it can be used.
+
+The primary license is the one that governs how our code can be used,
+modified, and shared, which is described more below. It is debated
```

Comment:
Who debates this? This could be revised to say "there are two ways of contributing to a project that ensures the code remain open source".